itests/qtest-spark/pom.xml

```diff
@@ -417,5 +417,53 @@
       </plugin>
     </plugins>
   </build>
-
+    <profiles>
```

Comment:
Please use 2 spaces as indentation, to keep consistency

Reply:
I don't know why my editor switched to 4 spaces...